src/ClassMapGenerator.php

```
@@ -216,10 +216,6 @@ private function filterByNamespace(array $classes, string $filePath, string $bas
         $realSubPath = substr($realSubPath, 0, $dotPosition === false ? PHP_INT_MAX : $dotPosition);
 
         foreach ($classes as $class) {
-            // silently skip if ns doesn't have common root
```

This check was explicitely here, but looking at the code it can be handled as a rejectedClass instead as it gets silently ignored anyway if at least one valid class is found. Then it can be used when checking with --strict-psr. What do you think?